### ISSUE TITLE
Dlls for both Big Data and Low Trust versions 1.1.0+ are not signed

### ISSUE THREAD

**Issue by amk334** [stephan-tolksdorf/fparsec#53]
The 1.1.0+ versions are no longer getting signed and they are getting flagged by Windows with unknown publisher.  I can't actually update to the latest version with this issue due to highly sensitive security checks on my side.

![image](https://user-images.githubusercontent.com/11759464/76647368-b355d500-652a-11ea-9ee1-30de5420b034.png)

**Comment by stephan-tolksdorf** [comment]
I've checked with `sn -vf FParsec.dll` in the Visual Studio Command Prompt that the FParsec and FParsecCS `net45` assemblies for both FParsec NuGet Packages are strongly named. The `netstandard2.0` assembly is not strong-named and has never been.

**Comment by amk334** [comment]
Hmm, ok. Thanks for checking.

**Comment by ptasz3k** [comment]
Is there any reason for not signing `netstandard2.0` assemblies? We're running into problems similiar to described in this [issue](https://github.com/accord-net/framework/issues/1338). I've referenced it here, as it contains reasonig for not mixing signed and not signed assemblies in nuget package.

We've separated huge codebase that depends on fparsec to `netstandard2.0` to share it between linux `netstandard2.0` server and windows wpf based application. Unfortunately, FParsec in wpf app tries to load not-signed FParsecCS assembly, but nuget puts `net45` strongly named assembly into application folder.

In effect we're experiencing `"Could not load file or assembly 'FParsecCS, Version=1.1.1.0, PublicKeyToken=null' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)"` error.
